modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomValidation.path

```diff
@@ -40,6 +40,11 @@
 		<td>//input[contains(@class,'toggle-switch-check')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>OBJECT_BASIC_INFO_TAB</td>
+		<td>//button[@class=('nav-link btn btn-unstyled') and contains(.,'Basic Info')]</td>
+		<td></td>
+	</tr>
```

Comment:
See https://github.com/timpak/liferay-portal/pull/252/commits/8855bac59942c02b2c7e6d49377596098db3f65a and https://github.com/timpak/liferay-portal/pull/252/commits/ce1b894df6bb4cfe4515aa5976fdd81f71ac5fa4.

When possible, we should try and use existing paths instead of creating new ones. Most of the UI elements in portal have generic paths in Poshi, especially if they're common across different features. In this case, we're able to use one generic macro for both tabs. There may be a generic gotoTab macro out there but we can implement that in a future refactor.